docs/README.md

```diff
@@ -6,3 +6,4 @@ This document briefly explains how you can get started with Scalar DB with a sim
 ## Getting Started
 ### [Electronic Money Application with Scalar DB on Cassandra](getting-started-with-cassandra.md)
 ### [Electronic Money Application with Scalar DB on Cosmos DB](getting-started-with-cosmosdb.md)
+### [Electronic Money Application with Scalar DB on Dynamo DB](getting-started-with-dynamodb.md)
```

Comment:
```suggestion
### [Electronic Money Application with Scalar DB on DynamoDB](getting-started-with-dynamodb.md)
```

Reply:
Thank you for your suggestions.
Document updated accordingly.